src/Paket.Bootstrapper/Paket.Bootstrapper.csproj

```diff
@@ -33,6 +33,7 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="FSharp.Core, Version=4.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
```

Comment:
this is what I wanted to avoid. the bootstrapper was supposed to be tiny. now we would have to merge a bg lib into it.